Test/civl/inductive-sequentialization/paxos/Paxos.bpl

```diff
@@ -79,7 +79,7 @@ function {:inline} IsDisjoint(ns1:NodeSet, ns2:NodeSet) : bool {
 // MaxRound(r, ns, voteInfo) returns the highest round less than r that some node in ns voted for.
 // If no node in ns has voted for a round less than r, then it returns 0.
 function MaxRound(r: Round, ns: NodeSet, voteInfo: [Round]Option VoteInfo): int;
-axiom (forall r: Round, ns: NodeSet, voteInfo: [Round]Option VoteInfo :: { MaxRound(r, ns, voteInfo) }
```

Comment:
Do you understand why verification is quicker with this trigger (and the other one in the invariant) removed? Are there more specific patterns that the solver finds to trigger on?

Reply:
I removed the trigger because it seemed silly and likely to be inferred automatically by the solver.  That it makes verification faster is news to me. Did you measure the times before and after?

Reply:
I tried `is.sh` which went from 5.4 seconds down to to 3.4 seconds. But now I also tried `ref.sh` which goes from 1.9 seconds up to 2.7 seconds. Well, still a net win :)